src/main/features/eligibility/store.ts

```diff
@@ -12,8 +11,8 @@ export const cookieName = 'eligibility-check'
 
 export class CookieEligibilityStore {
   read (req: express.Request, res: express.Response): Eligibility {
-    const cookie: string = new Cookies(req, res).get(cookieName)
-    return new Eligibility().deserialize(cookie !== undefined ? JSON.parse(cookie) : undefined)
+    const cookie: string = req.cookies[cookieName]
```

Comment:
@robertparkinson, I disabled cookie signing as that does not bring much value and it complicates route testing.